modules/cms-kit/src/Volo.CmsKit.Common.Application.Contracts/Volo/CmsKit/Contents/PageDto.cs

```diff
@@ -1,10 +1,8 @@
-﻿using System.Collections.Generic;
-using System;
+﻿using System;
 using Volo.Abp.Application.Dtos;
 
 namespace Volo.CmsKit.Contents;
 
-[Serializable]
```

Comment:
All DTOs should be serializable as declared in the best practices document: https://docs.abp.io/en/abp/latest/Best-Practices/Data-Transfer-Objects
You added `[Serializable]` to `DefaultContentDto` above, which is a good thing, but why we are removing this?

Reply:
Because it's inherited from `EntityDto`. `EntityDto` already provides `[Serializable]`.

Reply:
[Serializable] attribute is not inherited. So, EntityDto's serializable attribute doesn't make derived types serializable.